text/0517-io-os-reform.md

```diff
@@ -596,13 +616,17 @@ method, clients of `Read` can easily recover the `push` method.
 ### `Write`
 [Write]: #write
 
-The `Writer` trait is cut down to even smaller size:
+The `Writer` trait is cut down to even smaller size, and `flush` is taken
```

Comment:
note that this is an edit of a previous RFC which removed quite a few methods from `Write`. The statement you're commenting was preserved from that one.